README.md

```diff
@@ -30,7 +30,7 @@ To use this tool, make sure:
 Use the following command to download and install this tool:
 
 ```shell
-go install github.com/google/go-licenses@latest
+go get -tool github.com/google/go-licenses/v2@latest
```

Comment:
@Bobgy I doubt this is correct. Using Go 1.25, I ran your command and got this result:

```
$ go get -tool github.com/google/go-licenses/v2@latest
go: go.mod file not found in current directory or any parent directory.
        'go get' is no longer supported outside a module.
        To build and install a command, use 'go install' with a version,
        like 'go install example.com/cmd@latest'
        For more information, see https://golang.org/doc/go-get-install-deprecation
        or run 'go help get' or 'go help install'.
```

Reply:
Thanks for reporting that! So what's the latest canonical way to install in go?

Reply:
is https://github.com/google/go-licenses/pull/336 right?

Reply:
I guess so.